doc/user/using-java9.md

```diff
@@ -0,0 +1,37 @@
+# Using TruffleRuby with JDK 9
+
+TruffleRuby is designed to be run with a JVM that has the Graal compiler. The
+easiest way to do this is to use the GraalVM, which includes JDK 8, the Graal
+compiler, and TruffleRuby, all in one package.
+
+It is also possible to run on Java 9, but we do not support this and at the
+moment we only document it for interest.
+
+To run on Java 9, Graal needs to be built with Java 9 and there is no binary
+distribution of GraalVM based on Java 9 so you will need to
+[build Graal yourself](../contributor/building-graal.md), using Java 9 as the
+JDK. TruffleRuby itself needs to be built with Java 8 as normal.
+
+```
+$ GRAAL_REPO=...path to graal repository built with JDK 9...
```

Comment:
There should be a `export` in front here.

Reply:
@chrisseaton Did you miss the comment above?

Reply:
You meant in front of `JAVACMD` didn't you? I fixed that one.

Reply:
But also in front of `GRAAL_REPO`.
I thought otherwise it would not be set for later commands,
but actually it seems at least ZSH and Bash automatically export if there is no command following:
```bash
$ FOO=b
$ echo $FOO
b

# But
$  FOO=a some_command
$ echo $FOO

```
I think it's still good to have the explicit `export` to express the intent that it will be used later.

Reply:
But it doesn't need to be exported - it's only used in the following command.

Reply:
Ah, so the explicit `export` means the env var will be available to child processes, and without, only to the current shell process. Then the current version seems good, just confusing to me as I did not know these fine details.